docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -42,8 +60,256 @@ then try adding ``--ssh-key-file ~/.ssh/<name>``
+  
+  * Set the ``spec.selector.app`` to the value set in ``mdb-instance-name`` in
+    the ConfigMap followed by ``-ss``. For example, if the value set in the
+    ``mdb-instance-name`` is ``mdb-instance-0``, set  the
+    ``spec.selector.app`` to ``mdb-instance-0-ss``.
+  
+  * Start the Kubernetes Service:
+
+    .. code:: bash
+
+       $ kubectl --context k8s-bdb-test-cluster-0 apply -f mongodb/mongo-svc.yaml
+
+
+Step 7: Start the BigchainDB Kubernetes Service
+-----------------------------------------------
+
+  * This configuration is located in the file ``bigchaindb/bigchaindb-svc.yaml``.
+
+  * Set the ``metadata.name`` and ``metadata.labels.name`` to the value
+    set in ``bdb-instance-name`` in the ConfigMap above.
+  
+  * Set the ``spec.selector.app`` to the value set in ``bdb-instance-name`` in
+    the ConfigMap followed by ``-dep``. For example, if the value set in the
+    ``bdb-instance-name`` is ``bdb-instance-0``, set  the
+    ``spec.selector.app`` to ``bdb-instance-0-dep``.
+
+  * Start the Kubernetes Service:
+
+    .. code:: bash
+
+       $ kubectl --context k8s-bdb-test-cluster-0 apply -f bigchaindb/bigchaindb-svc.yaml
+
+
+Step 8: Start the NGINX Kubernetes Deployment
+---------------------------------------------
+
+  * NGINX is used as a proxy to both the BigchainDB and MongoDB instances in
+    the node. It proxies HTTP requests on port 80 to the BigchainDB backend,
+    and TCP connections on port 27017 to the MongoDB backend.
+
+  * As in step 4, you have the option to use vanilla NGINX or an OpenResty
+    NGINX integrated with 3scale API Gateway.
+
+Step 8.1: Vanilla NGINX
+^^^^^^^^^^^^^^^^^^^^^^^
+  
+  * This configuration is located in the file ``nginx/nginx-dep.yaml``.
+    
+  * Set the ``metadata.name`` and ``spec.template.metadata.labels.app``
+    to the value set in ``ngx-instance-name`` in the ConfigMap followed by a
+    ``-dep``. For example, if the value set in the ``ngx-instance-name`` is
+    ``ngx-instance-0``, set the fields to ``ngx-instance-0-dep``.
+
+  * Set ``MONGODB_BACKEND_HOST`` env var to
+    the value set in ``mdb-instance-name`` in the ConfigMap, followed by
+    ``.default.svc.cluster.local``. For example, if the value set in the
+    ``mdb-instance-name`` is ``mdb-instance-0``, set the
+    ``MONGODB_BACKEND_HOST`` env var to
+    ``mdb-instance-0.default.svc.cluster.local``.
+    
+  * Set ``BIGCHAINDB_BACKEND_HOST`` env var to
+    the value set in ``bdb-instance-name`` in the ConfigMap, followed by
+    ``.default.svc.cluster.local``. For example, if the value set in the
+    ``bdb-instance-name`` is ``bdb-instance-0``, set the
+    ``BIGCHAINDB_BACKEND_HOST`` env var to
+    ``bdb-instance-0.default.svc.cluster.local``.
+    
+  * Set ``MONGODB_FRONTEND_PORT`` to 27017, or the port number on which you
+    want to expose MongoDB service.
+    
+  * Set ``BIGCHAINDB_FRONTEND_PORT`` to 80, or the port number on which you
+    want to expose BigchainDB service.
```

Comment:
I wonder if we should also mention the `BIGCHAINDB_WS_FRONTEND_PORT`? (Both here and in Step 8.2)

As an alternative, why talk about changing those frontend ports at all? If they do, they'll also have to change them in some other places, otherwise the system won't work. My inclination is just to keep them on rails and only tell them to change the things that they must, i.e. omit all mentions of changing `*_FRONTEND_PORT` (both in 8.1 and 8.2).